controllers/badge.js

```diff
@@ -20,6 +20,20 @@ const getBadges = async (req, res) => {
   }
 }
 
+const getUserBadges = async (req, res) => {
+  try {
+    const allUserBadges = await badgeQuery.fetchUserBadges(req.params.username)
+    return res.json({
+      message: 'User badges returned successfully!',
+      userBadges: allUserBadges
```

Comment:
You can add a check for userbadge length and can print similar messages

Reply:
Yes I'll add that.